CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### 💡 Enhancements 💡
```

Comment:
We now have a new way of adding changelog entries. Check the contribution guidelines, but in short, just add a new file to ./unrelesed following the pattern used by the other files in there.

Reply:
Looks good, just one more question: don't we have an issue tracking this? I thought I've seen an issue open for this before.